phyloref.ofn

```diff
@@ -67,6 +70,15 @@ AnnotationAssertion(rdfs:seeAlso :newick_expression <https://en.wikipedia.org/wi
 #   Object Properties
 ############################
 
+# Object Property: :apomorphy (uses apomorphy)
+
+AnnotationAssertion(dc:description :apomorphy "Relates the subject to an apormorphy it uses for defining a clade. The subject may be a phyloreference, or a Phyx file (which is a container for the structured data making up the phyloreference).
```

Comment:
I don't think we ever use a Phyx file as a subject for `phyloref:apomorphy` -- we always use it to link a phyloreference to a phenotype. What's the case for doing that?

Reply:
My misunderstanding. I thought you need this for the container that the Phyx file is at the outer level.